bitnami/keycloak/templates/secret-external-db.yaml

```
@@ -10,8 +10,8 @@ metadata:
   name: {{ printf "%s-externaldb" .Release.Name }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- if or .Values.commonAnnotations .Values.externalDatabase.annotations }}
+  annotations: {{- include "common.tplvalues.merge" ( dict "values" (list .Values.commonAnnotations .Values.externalDatabase.annotations) "context" $ ) | nindent 4 }}
```

This is just a detail, but wouldn't it make more sense to swap the order in the objects to merge so the `externalDatabase.annotations` (more specific) take precedence over the common ones?


```suggestion
  annotations: {{- include "common.tplvalues.merge" ( dict "values" (list .Values.externalDatabase.annotations .Values.commonAnnotations) "context" $ ) | nindent 4 }}
```

---

Fixed :+1: